treeherder/extract/__init__.py

```diff
@@ -0,0 +1,7 @@
+import os
+import sys
+
+VENDOR_PATH = __file__.replace("__init__.py", "").rstrip(os.sep)+os.sep+"vendor"
+
+if VENDOR_PATH not in sys.path:
+    sys.path.append(VENDOR_PATH)
```

Comment:
The `isort` import problems come from here:  The idea is to add the `vendor` sub-directory only when we are call ing extraction code